Makefile

```diff
@@ -5,12 +5,9 @@ ifeq (${TIMESTAMP}, 1)
   SHELL = ./hack/timestamps.sh
 endif
 
-all:
-	hack/dockerized "export BUILD_ARCH=${BUILD_ARCH} && DOCKER_PREFIX=${DOCKER_PREFIX} DOCKER_TAG=${DOCKER_TAG} IMAGE_PULL_POLICY=${IMAGE_PULL_POLICY} VERBOSITY=${VERBOSITY} ./hack/build-manifests.sh && \
-	    hack/bazel-fmt.sh && hack/bazel-build.sh"
+all: format bazel-build manifests
```

Comment:
Isn't the order the opposite of what was here before? IIUC the current order is `manifests bazel-build format`

Reply:
Looks like the current order is `manifests format bazel-build` but yes, you're right, my PR does move manifests to the end.

TBH I don't see how the original order works, since manifests include shasums of components built by bazel-build...
The thing is we usually use `cluster-sync` to deploy, which rebuilds the manifests.
We could test this by doing `make && make push` to dockerhub and then use the CR and operator yamls from `_out` to deploy, see if that works.

Either way, I think building the manifests last make more sense, but if there are concerns I'm happy to keep the original order.

Reply:
Yup, makes a lot of sense. I'm convinced :)